app/Support/ContributorSync.php

```
@@ -0,0 +1,150 @@
+            }
+        }
+
+        if ($resolved === []) {
+            return $created;
+        }
+
+        $insert = $db->prepare(
+            'INSERT IGNORE INTO libri_autori (libro_id, autore_id, ruolo) VALUES (?, ?, ?)'
+        );
+        if ($insert === false) {
+            throw new \RuntimeException('Unable to prepare contributor link insert: ' . $db->error);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '140', column '23'). (undefined)

(MissingImport)